lib/vrl/stdlib/src/ip_ntop.rs

```diff
@@ -0,0 +1,110 @@
+use std::net::IpAddr;
+
+use vrl::prelude::*;
+
+fn ip_ntop(value: Value) -> Resolved {
+    let value = value.try_bytes()?;
+
+    match value.len() {
+        4 => {
+            let bytes: [u8; 4] = value[..].try_into().unwrap();
```

Comment:
Not a big deal, but it's a good practice to use `expect()` rather than `unwrap()` to make the invariant clearer. Here it could be something like: `.expect("invalid length")`.

Reply:
I went to check what `expect()` does and I agree is nicer! I will update it following your advice.
Thanks for the kind words and feedback! And happy to help you guys.